ui/store/store.js

```diff
@@ -13,6 +13,6 @@ export default function configureStore(initialState) {
   return createStore(
     rootReducer,
     initialState,
-    composeEnhancers(applyMiddleware(thunkMiddleware)),
+    composeEnhancers(applyMiddleware(thunkMiddleware.withExtraArgument(t))),
```

Comment:
There is this extra argument used :thinking:

Reply:
failed experiment! will remove. Thought it would be cool to give our state trees the ability to use the translation method instead of storing translation keys in state. Didn't pan out due to needing context and react to call this method.

Will remove!

Reply:
removed in latest